server/planning/planning_locks.py

```diff
@@ -53,32 +46,49 @@ class PlanningLockRepos(Enum):
 ]
 
 
-bp = Blueprint("planning_locks", __name__)
+class PlanningLocksParams(BaseModel):
+    repos: Annotated[list[PlanningLockRepos], Field(default=DEFAULT_REPOS)]
```

Comment:
`default` here is mutable, therefor it would be safer to use default_factory to provide a copy of the `DEFAULT_REPOS` variable